README.md

```diff
@@ -209,6 +210,7 @@ Tests should be added for each new functionality. Be sure to run tests before pr
 
 ## Changelog
 
+  - **Jun. 25, 2018** - Tasks can now be undone
```

Comment:
Is it possible to follow same naming convention as commit (the imperative present tense), sth like "Add option to undone a task", or "Add ability to undone a task" ?

Reply:
"Undone" is an adjective (or the past participle of the verb "undo").. I am changing the wording to "Add ability to mark tasks as undone". I hope that will do :)